src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.cc

```diff
@@ -153,7 +153,10 @@ static void grpc_ares_request_unref(grpc_ares_request* r) {
   /* If there are no pending queries, invoke on_done callback and destroy the
      request */
   if (gpr_unref(&r->pending_queries)) {
-    grpc_cares_wrapper_address_sorting_sort(*(r->lb_addrs_out));
+    grpc_lb_addresses* lb_addrs = *(r->lb_addrs_out);
+    if (lb_addrs) {
+      grpc_cares_wrapper_address_sorting_sort(lb_addrs);
```

Comment:
nit: `s/if(lb_addrs)/if(lb_addrs != nullptr)`

LGTM, and I can try to add those, thanks for pointing out

Reply:
fixed.